ebpf/tracer.go

```diff
@@ -154,6 +163,13 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// shouldSkipConnection returns whether or not the tracer should ignore a given connection:
+//  • Local DNS (*:53) requests if configured (default: true)
+func (t *Tracer) shouldSkipConnection(conn *ConnectionStats, direction ConnectionDirection) bool {
```

Comment:
Do you want to check for UDP here?

Reply:
DNS calls can be TCP as well

Reply:
https://en.wikipedia.org/wiki/Domain_Name_System#DNS_protocol_transport